### ISSUE TITLE
Decide on project management (membership) solution

### ISSUE THREAD

**Issue by pontus** [NBISweden/aida-data-hub#787]
Some settings for PI/trusted delegate will clearly make sense to have on platform (e.g. knobs for services, security levels et.c.). But we need to decide how we want the project management/membership editing experience to be (e.g. adding/removing e-mails on platform, perun groups, some other external service)

**Comment by eryl** [comment]
We're contacting NAISS people to hear about whether SUPR would work for us

**Comment by pontus** [comment]
I have reached out to Niclas about things that may need to be discussed.

**Comment by yohell** [comment]
Now in contact with doers. Talking is easy, making decision is hard. I'd give a time estimate of 1, blocked by another card for discussions (which may not happen until after christmas?).

**Comment by yohell** [comment]
Status? Still needed? Icebox?

**Comment by yohell** [comment]
Status is unchanged, I will follow up SUPR status with NAISS next week. Also, there is a similar system called waldur.com that is favored by EuroHPC so will most likely be used by both Arrhenius and Mimer, and might be worth looking into. Either way, this is lower priority than having REMS in production on DSP.
